demonstrations/tutorial_how_to_use_arithmetic_operators.metadata.json

```
@@ -0,0 +1,60 @@
+{
+    "title": "How-to use Quantum Arithmetic Operators",
+    "authors": [
+        {
+            "id": "gmlejarza"
+        },
+        {
+            "id": "KetPuntoG"
+        }
+    ],
+    "dateOfPublication": "2024-10-01T00:00:00+00:00",
+    "dateOfLastModification": "2024-10-01T00:00:00+00:00",
+    "categories": [
+        "Quantum Computing",
+        "Algorithms"
+    ],
```

Is this supposed to be a How-to?
```suggestion
    "categories": [
        "Quantum Computing",
        "Algorithms",
        "How-to"
    ],
```